index.js

```diff
@@ -236,7 +232,7 @@ class VeseyncPlugPlatform {
                 return this.client.turnDevice(device, "off");
             }
 
-            if (device.status == 'off' && powerState == true) {
+            if ((device.status == 'off' || !device.status) && powerState == true) {
```

Comment:
this device.status is always null... probably why the home bridge dashboard is not showing the power status